src/Tribe/Service_Providers/Context.php

```diff
@@ -171,14 +173,14 @@ static function () {
 						Tribe__Context::QUERY_VAR   => 'tribe_remove_date_filters',
 					],
 				],
-				'event_display_mode'   => [
+				'event_display_mode' => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'event_display_mode'" and double arrow, but found 1.

<sub>hash: 38c222b7d773c7423ff932108748c7f2</sub>